src/components/MapController/MapController.js

```diff
@@ -12,37 +13,68 @@ import styles from './MapController.css';
 
 var MapController = React.createClass({
 
-    propTypes: {
-        onSetArea: React.PropTypes.func.isRequired,
-    },
+  propTypes: {
+    variable: React.PropTypes.string,
+    meta: React.PropTypes.array,
+    onSetArea: React.PropTypes.func.isRequired,
+  },
```

Comment:
`timeidx` and `wmstime` and are contained in the state. Now documented in 0e7dea3.